ghost/core/core/server/services/segment/DomainEventsAnalytics.js

```diff
@@ -0,0 +1,40 @@
+const _ = require('lodash');
+const logging = require('@tryghost/logging');
+const DomainEvents = require('@tryghost/domain-events');
+const {MilestoneCreatedEvent} = require('@tryghost/milestones');
+
+module.exports = class DomainEventsAnalytics {
+    #analytics;
+    #trackDefaults;
+    #prefix;
+    #sentry;
+
+    constructor(deps) {
+        this.#analytics = deps.analytics;
+        this.#trackDefaults = deps.trackDefaults;
+        this.#prefix = deps.prefix;
+        this.#sentry = deps.sentry;
+    }
+
+    async #handleMilestoneCreatedEvent(type, event) {
+        if (type === MilestoneCreatedEvent
```

Comment:
I don't think you need this check?

Reply:
Yeah, will have to touch that again, once we might send a 'StripeLiveEnabled' event or similar